scss/components/_state-screen.scss

```diff
@@ -129,11 +129,19 @@
     }
   }
 
-  .oo-action-icon:focus {
+  .oo-action-icon:focus {    
+    @include highlight-tab();
+  }
+
+  .oo-action-icon:hover {
```

Comment:
I think it might be better to remove this hover style since it wasn't there on the previous version.